### ISSUE TITLE
New API: fast colors

### ISSUE THREAD

**Issue by antonmedv** [charmbracelet/lipgloss#81]
Hi,

I'm building a new tool that requires a lot of Render calls (right now I prerender every line for a viewbox). The cpu profile shows lipgloss.Render func is taking lots of time:
<img width="1560" alt="Снимок экрана 2022-04-14 в 18 13 49" src="https://user-images.githubusercontent.com/141232/163430657-02383b1c-4b61-41d0-85da-7377ae239282.png">

But I use only colors.  Is it possible to add new methods, which allow to access only color functionality (no padding, borders, etc.)?  

Thanks!

**Comment by meowgorithm** [comment]
That's a great idea; we can totally optimize for that. Probably better than adding a new method would be for `Render` to simply short circuit if the style only contains ANSI sequences (colors, bold, underline, etc).

**Comment by meowgorithm** [comment]
This could be further optimized, but what about something roughly like this: https://github.com/charmbracelet/lipgloss/commit/d92e7196bc88063dd54f230d59fd0000638e038e? This will effectively exit the render early if there are only text colors and text styles to render.

**Comment by antonmedv** [comment]
I can try check this branch.

**Comment by antonmedv** [comment]
Seems better, but still:
<img width="1231" alt="Снимок экрана 2022-04-14 в 23 17 56" src="https://user-images.githubusercontent.com/141232/163477986-07b4c718-c460-4b6d-9b4a-374408505dde.png">

**Comment by meowgorithm** [comment]
Yeah, it’s an improvement but we can do better. Have some ideas, will look into this some more.
